solutions/standard/catalogValidationValues.json.template

```diff
@@ -0,0 +1,10 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "region": "us-east",
+  "resource_group_name": $PREFIX,
+  "existing_mq_capacity_crn": $HPCS_US_SOUTH_CRN,
```

Comment:
Ah I'm going to have to add support to the catalog pipeline to pull the MQ capacity CRN here - I created an internal issue